src/website/app/pages/ComponentStatus/componentStatus.md

```diff
@@ -23,20 +23,21 @@ Statuses:
 
 | Component | Status |
 |---|---|
+| NumericInput | [planned] |
```

Comment:
Also, what is determining the order of the planned work?  For example, I would guess that we would build TextInput prior to NumericInput.

Reply:
I just grouped each section (planned, available, etc) alphabetically so we have some wiggle room.